### PR TITLE
English Docs

### PR DESCRIPTION
**Pull request by medienlampe**
Привет!

I translated the documentation so this cool library might be found and used by more people. :)
Please let me know of you want something changed.

### CODE REVIEW

File: radiacode-examples/README.md [hunk 1]
```diff
@@ -1,25 +1,22 @@
-# Примеры использования библиотеки
-
-Устанавливаются с пакетом если указать `pip install radiacode[examples]` (вместо `pip install radiacode`)
-
-У каждого примера есть справка по `--help`
+# RadiaCode Library - Examples
 
+These example projects are installed with the libary if you specify `pip install radiacode[examples]` (instead of `pip install radiacode`).
```
Comment (cdump) [inline radiacode-examples/README.md:3]: libary -> library

**Review by cdump — CHANGES_REQUESTED** [review]
Hi! Thank you for your translation, I found one small typo.

I think that we still need Russian documentation in README_ru.md and link to it at the top of English README.md, not all Russian users can understand English - I can fix it myself after merge your MR.

File: radiacode-examples/README.md (hunk 1 above)
Comment (medienlampe) [reply radiacode-examples/README.md]: Thx, it's fixed now.

**Comment by medienlampe** [comment]
Hey,

that makes sense!
I added the russian versions back in to make it simpler for you.
Sadly, I'm not able to link it by myself, as I don't speak russion.

Also, the change has been made.

Have a great day!

**Comment by cdump** [comment]
Merged, I've added links to Russian docs in additional commit

**Pull request merged**